odoo/tools/misc.py

```diff
@@ -1202,7 +1202,7 @@ def get_lang(env, lang_code=False):
     :param str lang_code: the locale (i.e. en_US)
     :return res.lang: the first lang found that is installed on the system.
     """
-    langs = [code for code, _ in env['res.lang'].get_installed()]
+    langs = [code for code, _ in env['res.lang'].with_context(lang='en_US').get_installed()]
```

Comment:
The context in the returned records may still be invalid

Reply:
what do you suggest?

Reply:
always `return env['res.lang'].with_context(lang=lang)._lang_get(lang)`?
It should be bug free for the current use cases, but it makes the function more tricky